### ISSUE TITLE
After killing Icasaracht, the game froze and stopped moving.

### ISSUE THREAD

**Issue by Lzw104522773** [The-Gate-Project/IWD1_EET#24]
After killing Icasaracht, the game froze and stopped moving. My friend said to get very close to the Soul Gem and kill Icasaracht in order for the game to continue smoothly.

![F7E6AAA98B68F65354741D92EC152B13](https://github.com/user-attachments/assets/1725f131-a767-4492-99aa-3bcfc1df79ae)

**Comment by abalabokhin** [comment]
The animation continued working in this case, right? If yes, it looks like cutscene started but EndCutSceneMode() was never called.

**Comment by Lzw104522773** [comment]
yes,in this case,the animation continued working,there is also sound.

**Comment by Lzw104522773** [comment]
it need to kill Icasaracht at a specific location， then the game can continue.

**Comment by 11jo** [comment]
Should be good [now](https://github.com/The-Gate-Project/IWD1_EET/commit/90711e0905db495e268c785b275b4b24faa5fede).
